docs/en-us/manual/introduction/tools.md

```diff
@@ -36,8 +36,14 @@ Only 16:9 aspect ratio videos with a resolution of 720p or higher are supported.
 
 ## Gacha
 
-**真的是用你的号抽你的卡 真的是用你的号抽你的卡 真的是用你的号抽你的卡**
-**真的是用你的号抽你的卡 真的是用你的号抽你的卡 真的是用你的号抽你的卡**
-**真的是用你的号抽你的卡 真的是用你的号抽你的卡 真的是用你的号抽你的卡**
+**It really uses your account to draw your cards! It really uses your account to draw your cards!**
+**It really uses your account to draw your cards! It really uses your account to draw your cards!**
+**It really uses your account to draw your cards! It really uses your account to draw your cards!**
 
-牛牛航空没报上名？没关系！现在每个人都有自己的牛牛航空！并且在抽卡的过程中还会截图并显示在界面中央！
+Didn't get a spot in [NiuNiu Airlines](https://www.bilibili.com/opus/926129394412421126)? No problem! Now everyone has their own NiuNiu Airlines! And it will take screenshots and display them in the center of the interface during the draw!
+
+<!-- markdownlint-disable -->
+
+_(WTF is NiuNiu Airlines)_
```

Comment:
I just noticed lmao, is this from GPT itself?

Reply:
no it's written by myself

i just wonder

Reply:
Oh, I thought XD.
(I also wonder, since I don't understand chinese I just ignore most of the times these things as I think they are idioms or very known memes in China)

Reply:
xx航空 means UPs(in China on bilibili, same as Youtubers) use audience's account to gacha
牛牛 means pallas whose ethnicity is 丰蹄(Forte in English maybe i can't find translation) which prototype is bull or cow

MAA has a channel on bilibili that provide VUP(Virtual UP, same as VTB (Virtual Youtuber)) live broadcast, and there's a cute girl live on it. [channel link](https://space.bilibili.com/3493274731940507)
it's a pop culture thing in anime mobile game live, maybe, cuz many broadcasters do this

btw, there's a saying that is "airline crashing", meaning cannot drawn current character in certain times
and "flight arrivals" meaning succeed

Reply:
Ok I had no idea we have a MAA Vtuber